common/history/countries/kor - korea.txt

```diff
@@ -1,7 +1,10 @@
 ﻿COUNTRIES = {
 	c:KOR = {
 		effect_starting_technology_tier_4_tech = yes
-	
+		
+		add_technology_researched = urban_planning
+		add_technology_researched = law_enforcement
+
```

Comment:
still needs starting laws, the list we have isn't all laws.

Reply:
Would a `effect_starting_politics_traditional = yes` statement work?

Reply:
yes that one is needed

Reply:
above the adjustments we make